web/src/main/java/org/mskcc/cbio/oncokb/controller/EvidenceController.java

```diff
@@ -231,7 +252,7 @@ ResponseEntity updateVUS(@ApiParam(value = "hugoSymbol", required = true) @PathV
                 knownEffect = oncogenicity.getOncogenic();
             }
         }
-
+        // save newly added evidence    
         if (evidences.isEmpty()) {
```

Comment:
Remember to duplicate oncogenic evidence based on number of mutations in the string